sdk/python/kfp/v2/dsl/_pipeline.py

```diff
@@ -108,5 +108,9 @@ def add_op(self, op: _container_op.BaseOp) -> str:
     op_name = _naming._make_name_unique_by_adding_index(op.human_name,
                                                         list(self.ops.keys()),
                                                         ' ')
+    if op_name == '':
```

Comment:
For my education: when will the op_name become empty, even after the first call of `_make_name_unique_by_adding_index`? Is it because `op.human_name` is empty?

Reply:
If so then consider changing line 108 to the following
```
op_name = _naming._make_name_unique_by_adding_index(op.human_name or 'task',
```

Reply:
Removed this code. It could happen in v1 where it sanitize the name by removing special characters.
But with v2, we don't have the sanitization step, and special characters are allowed for task name.